yarn-project/aztec/src/bin/index.ts

```diff
@@ -41,5 +41,7 @@ async function main() {
 main().catch(err => {
   debugLogger.error(`Error in command execution`);
   debugLogger.error(err + '\n' + err.stack);
-  process.exit(1);
+  // See https://nodejs.org/api/process.html#processexitcode
```

Comment:
I changed this because I saw logs where this was getting hit but the process wasn't exiting. 

> Rather than calling process.exit() directly, the code should set the process.exitCode and allow the process to exit naturally by avoiding scheduling any additional work for the event loop
> If it is necessary to terminate the Node.js process due to an error condition, throwing an uncaught error and allowing the process to terminate accordingly is safer than calling process.exit().